docs/conf.py

```diff
@@ -327,6 +327,12 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
 
+epub_exclude_files = [
+    "_static/favicon.ico",
+    "_static/wagtail_colors_tables.txt",
+    "_static/wagtail_icons_table.txt",
+]
```

Comment:
Have you tried building the epub file locally? What happens to these files in the output?

An alternative may be to suppress the warning altogether, see https://github.com/sphinx-doc/sphinx/commit/6d900c34f12db0d21c581c58a5dd38ab49c8ea35 and https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-suppress_warnings

Reply:
OK, looks like this doesn't make a difference in the `.epub` output:
<img width="792" alt="image" src="https://github.com/user-attachments/assets/88b7d890-b12c-479f-a537-f6ca2663c079">

Looking at the warnings:

```
WARNING: unknown mimetype for robots.txt, ignoring
WARNING: unknown mimetype for _static/favicon.ico, ignoring
WARNING: unknown mimetype for _static/wagtail_colors_tables.txt, ignoring
WARNING: unknown mimetype for _static/wagtail_icons_table.txt, ignoring
```

Looks like they got ignored anyway. So I think we should just suppress the warning, so we don't have to keep updating this list every now and then.